modules/knative-serving-advanced-config.adoc

```diff
@@ -6,7 +6,8 @@
 [id="knative-serving-advanced-config_{context}"]
 = Advanced configuration options
 
-This section provides information about advanced configuration options for Knative Serving.
+You can install Knative Serving by using the default settings, or configure more advanced settings in the `KnativeServing` custom resource (CR). Configuring these advanced settings allows you to modify functionality such as use of certificates for your Knative Serving deployment, and the number of high availability replicas that are available.
```

Comment:
> ! allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

Maybe:

```suggestion
You can install Knative Serving by using the default settings or configure more advanced settings in the `KnativeServing` custom resource (CR). By configuring these advanced settings, you can modify functionality such as use of certificates for your Knative Serving deployment and the number of high availability replicas that are available.
```